src/assets/toolkit/styles/components/message.css

```diff
@@ -0,0 +1,38 @@
+/** @define Message */
+
+:root {
+  --Message-padding: var(--space-xs) 2.5em;
+  --Message-background: var(--color-gray);
+  --Message-offline-background: var(--color-orange);
+  --Message-offline-color: var(--color-white);
+  --Message-close-size: 1.4em;
```

Comment:
Did you try [our modular sizes](https://github.com/cloudfour/cloudfour.com-patterns/blob/master/src/assets/toolkit/styles/base/_config.css#L109-L113) and find that none of them worked?